src/index.html

```diff
@@ -2,11 +2,53 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        initial-scale=1.0"
+    >
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div
+      class="
+      calendar
+      calendar--start-day--sun
+      calendar--month-length--29"
+    >
```

Comment:
you've over-split stuff - one attribute - values should be in one line - then no error occurs
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```